docs/install_local.md

```diff
@@ -85,22 +93,20 @@ git fetch --all
 pip install -e .
 ```
 Note: if you want the newest updates to the WHOC repository, you can checkout the develop branch instead of the main branch.
-<!--
-# Other steps for era 5
-Now need to add a file called APIKEY which contains the API Key you'll find in your data.planetos account
 
-The instructions said to place it in the folder
-OpenOA/operational_analysis/toolkits
 
-But I found I also had to copy it down to here:
-/Users/pfleming/opt/anaconda3/envs/hercules/lib/python3.8/site-packages/operational_analysis/toolkits/
+# Running [Local]
 
-Col
--->
+## Using the run scripts
 
-# Running [Local]
+The simplest way to run hercules is to use the provided run scripts via:
+```
+bash run_script.sh
+```
+
+## Running manually
 
-To run locally using a standin for AMR-Wind, launch 3 separate 
+To run locally without the included runscript using a standin for AMR-Wind, launch 3 separate 
 terminals. In each, `conda activate` your hercules environment (`conda 
 activate hercules`). 
 
```

Comment:
I can't actually comment on the lines below this, because this PR doesn't change them, but they seem to assume the user has already declared a `HELICS_PORT` variable?

Reply:
Yeah, I'm not positive about this whole section.  In the same way as the runscript in each folder might be a little different, the manual version might also be a little different and the best approach would be just to enter the lines appearing in the runscript as they appear.  Possibly in 3 terminals, or using the & to unblock, maybe just shorten this section to something more like "to be sure, it's possible to enter the commands in the runscript directly..."?

Reply:
I think it makes sense to encourage people to use the run_script.sh . So, perhaps those instructions should come first. But then, maybe it is worth taking the example of one of the run_script.sh , and talking through what each line is doing (and explain that it is possible to run this from three separate terminals, if that's what the user would prefer to do)?